src/taskgraph/util/vcs.py

```diff
@@ -477,7 +480,10 @@ def get_commit_message(self, revision=None):
 
     def get_tracked_files(self, *paths, rev=None):
         rev = rev or "HEAD"
-        return self.run("ls-tree", "-r", "--name-only", rev, *paths).splitlines()
+        cmd = ["ls-tree", "-r", "--name-only", rev]
+        if paths:
+            cmd.extend(paths)
+        return self.run(*cmd).splitlines()
```

Comment:
To Julien's point, this isn't removing any empty strings:
```
>>> a = []
>>> [1,2,3,*a]
[1, 2, 3]

>>> a = ['']
>>> [1,2,3,*a]
[1, 2, 3, '']
```
If the idea is to filter out, you'd probably need something like `[p for p in paths if p]`

Reply:
Indeed.  (Also, it'd be good to understand/document why/how an empty string would come in here, naively that seems like it'd be a bug elsewhere.)

Reply:
I decided to strip out empty strings rather than make `graph_config.vcs_path` absolute to avoid causing issues elsewhere.